src/composables/transactionTokens.ts

```diff
@@ -61,7 +61,7 @@ export function useTransactionTokens({
     const isReceived = direction === TX_DIRECTION.received;
     const adapter = ProtocolAdapterFactory.getAdapter(protocol);
     const amount = (isAllowance)
-      ? toShiftedBigNumber(innerTx.value?.fee || 0, -adapter.getAmountPrecision())
+      ? toShiftedBigNumber(innerTx.value?.fee || 0, -adapter.coinPrecision)
```

Comment:
I think that 18 decimals is way to much for a displayed amount, is that really what we want? 
Or do we format this amount again in a later stage before showing it?